universal-application-tool-0.0.1/conf/application.conf

```
@@ -434,4 +434,4 @@ aws.s3.bucket=${?AWS_S3_BUCKET_NAME}
 aws.local.endpoint="http://localstack:4566"
 
 ## Feature flags
-cf.optional_questions = ${?CF_OPTIONAL_QUESTIONS}
\ No newline at end of file
+cf.optional_questions = "true" # ${?CF_OPTIONAL_QUESTIONS}
```

Should we have this to default to true, but allow override by environment variable to make this more easily runtime settable? e.g., this syntax seems to be used elsewhere
```suggestion
cf.optional_questions = true
cf.optional_questions = ${?CF_OPTIONAL_QUESTIONS}
```

Also, seems like the value here should ideally be boolean true instead of string "true" (even though I'm guessing the non-empty string evaluates to true, this would be more consistent with the rest of the file)?

---

Done.  Thank you for the thorough review throughout!